msc_pygeoapi/loader/aqhi_realtime.py

```diff
@@ -0,0 +1,492 @@
+        files_to_process.sort(key=os.path.getmtime)
+
+    for file_to_process in files_to_process:
+        loader = AqhiRealtimeLoader(conn_config)
+        result = loader.load_data(file_to_process)
+        if result:
+            click.echo(
+                'GeoJSON features generated: {}'.format(
+                    json_pretty_print(loader.items)
+                )
+            )
+
+
+@click.command()
+@click.pass_context
+@cli_options.OPTION_FILE()
+@cli_options.OPTION_ELASTICSEARCH()
+@cli_options.OPTION_ES_USERNAME()
+@cli_options.OPTION_ES_PASSWORD()
+@cli_options.OPTION_ES_IGNORE_CERTS()
+def add_regions(ctx, file_, es, username, password, ignore_certs):
+    """Add AQHI regions index"""
+
+    conn_config = configure_es_connection(es, username, password, ignore_certs)
+
+    # if no file path is given retrieve default aqhi_community.json file
+    # in msc_pygeoapi/resources
+    if file_ is None:
+        filepath = os.path.join(MSC_PYGEOAPI_BASEPATH, 'resources', 'aqhi_community.geojson')
+    else:
+        filepath = file_
+
+    loader = AqhiRealtimeLoader(conn_config)
+    package = loader.generate_regions(filepath)
+    loader.conn.submit_elastic_package(package, request_size=80000)
+
+    return True
+
+
+@click.command()
+@click.pass_context
+@cli_options.OPTION_DAYS(
+    default=DAYS_TO_KEEP,
+    help='Delete indexes older than n days (default={})'.format(DAYS_TO_KEEP),
+)
+@cli_options.OPTION_DATASET(
+    help='AQHI dataset indexes to delete.',
+    type=click.Choice(['all', 'forecasts', 'observations']),
+)
+@cli_options.OPTION_ELASTICSEARCH()
+@cli_options.OPTION_ES_USERNAME()
+@cli_options.OPTION_ES_PASSWORD()
+@cli_options.OPTION_ES_IGNORE_CERTS()
+@cli_options.OPTION_YES(prompt='Are you sure you want to delete old indexes?')
+def clean_indexes(ctx, days, dataset, es, username, password, ignore_certs):
+    """Delete old AQHI realtime indexes older than n days"""
+
+    conn_config = configure_es_connection(es, username, password, ignore_certs)
+    conn = ElasticsearchConnector(conn_config)
+
+    if dataset == 'all':
+        indexes_to_fetch = '{}*'.format(INDEX_BASENAME.format('*'))
+    else:
+        indexes_to_fetch = '{}*'.format(INDEX_BASENAME.format(dataset))
+
+    indexes = conn.get(indexes_to_fetch)
+
+    if indexes:
+        indexes_to_delete = check_es_indexes_to_delete(indexes, days)
+        if indexes_to_delete:
+            click.echo('Deleting indexes {}'.format(indexes_to_delete))
+            conn.delete(','.join(indexes))
+
+    click.echo('Done')
+
+
+@click.command()
+@click.pass_context
+@cli_options.OPTION_DATASET(
+    help='AQHI dataset indexes to delete.',
+    type=click.Choice(['all', 'forecasts', 'observations', 'regions']),
+)
+@cli_options.OPTION_ELASTICSEARCH()
+@cli_options.OPTION_ES_USERNAME()
+@cli_options.OPTION_ES_PASSWORD()
+@cli_options.OPTION_ES_IGNORE_CERTS()
+def delete_indexes(ctx, dataset, es, username, password, ignore_certs):
```

Comment:
See https://github.com/ECCC-MSC/msc-pygeoapi/blob/master/msc_pygeoapi/loader/bulletins_realtime.py#L207-L233 for example (which also includes index template deletion).